app/views/admin/works/_full_text.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="card bg-light mb-3">
```

Comment:
`works/_full_text` is not descriptive enough for what this is. It's really`oral_history_transcript.html.erb` maybe?

Reply:
Done.